rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/rascal/RascalLanguageServices.java

```diff
@@ -235,6 +233,36 @@ public InterruptibleFuture<ITuple> getRename(ITree module, Position cursor, Set<
         }, VF.tuple(VF.list(), VF.map()), exec, false, client);
     }
 
+    private ISourceLocation sourceLocationFromUri(String uri) {
```

Comment:
We should add this (in a different PR) to the `URIUtil` in rascal, and call it `assumeCorrectLocation` (to mirror the `assumeCorrect` in that helper class).

Reply:
https://github.com/usethesource/rascal/pull/2113

Reply:
tnx 👍🏼